lib/bounces.js

```diff
@@ -108,6 +108,24 @@ module.exports = function (log, error) {
           }
         }
 
+        // Log flow metrics if `flowId` and `flowBeginTime` specified in headers
+        const flowId = getHeaderValue('X-Flow-Id', message)
+        const flowBeginTime = getHeaderValue('X-Flow-Begin-Time', message)
+        if (flowId && flowBeginTime) {
+          const eventName = 'email.' + templateName + '.bounced'
+          const currentTime = Date.now()
+
+          // Matches structure emitted by https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js#L122
+          const flowEventInfo = {
+            event: eventName,
+            time: currentTime,
+            flow_id: flowId,
+            flow_time: (currentTime <= flowBeginTime) ? 0 : (currentTime - flowBeginTime)
```

Comment:
Alternatively, if we are confident that will never happen because `flowBeginTime` is ultimately the one we already validated on entry to the auth server, then just don't bother checking it all. But having the check like this sends a misleading message to future readers I think.